__tests__/app.test.js

```diff
@@ -65,3 +65,23 @@ describe("GET /api/articles/:article_id", () => {
     expect(response.body.msg).toBe("Bad Request");
   });
 });
+
+describe("GET /api/articles", () => {
+  it("200: responds with all of the articles without body property and with comment_count property", async () => {
+    const response = await request(app).get("/api/articles");
+    expect(response.status).toBe(200);
+    response.body.articles.map((article) => {
```

Comment:
We should really be checking the length of the array before this to ensure we're getting all of the articles back 👍